src/styles/main.scss

```diff
@@ -0,0 +1,30 @@
+.container {
+  --column-count: 1;
+
+  min-width: 1024px;
+  max-width: 1200px;
+  padding: 50px 40px 72px;
```

Comment:
The padding values seem to be arbitrary. Make sure that they adhere to the 4px/8px scale as per the code style rules. If these values are outside the scale, they should be justified by the design requirements.